### ISSUE TITLE
Clarification on cmd commands to get vagrant running, quicker.

### ISSUE THREAD

**Issue by simily00** [open-learning-exchange/open-learning-exchange.github.io#2060]
### Problem
I found it time consuming and confusing when I needed to get my vagrant environment back up and running. I found myself having to look back through the OLE support steps over and over again to see the commands I needed to get the environment(s) running. I think if the steps for accessing each community were in one place and highlighted, that it would make life easier for incoming intern prospects and much more efficient. I would apply this Quick Reference guide to the other environments as well: vagrant, Docker [...etc].

### Steps to reproduce the problem

N/A

### Screenshots

![easy start up_](https://user-images.githubusercontent.com/17802123/46251675-d19b4400-c40e-11e8-92aa-994dc7ed0e7c.jpg)

### Proposed solution

Please see screenshot

**Comment by nadaa** [comment]
Thanks @simily00, I don't think you need to start your virtual box manually, the command  `vagrant up` does this!

**Comment by simily00** [comment]
Hi @nadaa , thanks for commenting. Question, when you shutdown your computer for the day, and turn it back on to resume work, to get your Vagrant environment started, you reopen your terminal and simply put in: "vagrant up" and the environment begins running? 

I ask because I understand your point but I always get this error(when I ONLY use the "vagrant up" command: 
![screen shot 2018-09-30 at 2 54 30 pm](https://user-images.githubusercontent.com/17802123/46263218-416a0700-c4c1-11e8-9f40-5a23b4280eef.jpg)

**Comment by svlesiv** [comment]
Hi @simily00, I'm running Bell on MacBook Air 10.13.6. For that, I need to navigate to ole--vagrant-vi directory and type - vagrant up in the terminal. But I don't use Ubuntu on my Mac.

**Comment by lmmrssa** [comment]
@simily00 vagrant command needs to run from where you copied `ole--vagrant-vi` which means you need to be inside `ole--vagrant-vi` because you have **Vagrantfile** inside that folder. This step is same for all Operating systems.
You can instead add step to restart vagrant after you shutdown on same page

**Comment by nadaa** [comment]
@simily00, did you install vagrant on mac or you install it on Debian VM?. For I have Windows, and the command `vagrant up` runs at the startup.

**Comment by simily00** [comment]
@lmmrssa Okay I will add that step you've suggested.

**Comment by simily00** [comment]
@lmmrssa 
![easystartup_](https://user-images.githubusercontent.com/17802123/46385667-1f63b680-c672-11e8-8aeb-796515a04e4c.jpg)


better?

**Comment by inDepthh** [comment]
@simily00 I think that this picture needs some work. Visually, in my opinion, it does not look great. Indentation is off, the theme is different then everything else on the website (with the blue). The same thing can be replicated with a more simple layout and design that doesn't have this much redundancy, such as telling the user how to exit out of virtual box. If the user does not know how to do that, then he/she should not be getting a software dev internship. Overall, a good effort, but needs work. Hope I have been helpful

**Comment by simily00** [comment]
@inDepthh Thanks for your feedback! I obviously know the theme doesn't match, It was merely a visual idea not a proposed visual change. But, that does bring up a great point regarding theme, or lack thereof on the current site.

**Comment by YuserahN** [comment]
@simily00 I don't think that picture is necessary. I am not entirely sure how it is for Mac. But for Windows, all you need to do is open the terminal, cd to the folder where you installed vagrant, and type 'vagrant up' in the terminal. And make sure to use the halt command to pause it so it's not running while you're attempting to shut down your computer. 
In my opinion, this can be added as a little note, but it does need a visual.

**Comment by coder8102** [comment]
> @simily00 I don't think that picture is necessary. I am not entirely sure how it is for Mac. But for Windows, all you need to do is open the terminal, cd to the folder where you installed vagrant, and type 'vagrant up' in the terminal. And when you're done, make sure to use the halt command to pause it.
> In my opinion, this can be added as a little note, but it does not need a visual.

Yes. I agree with YuserahN that picture is not relevant for windows. I am using windows and all you need to do is to cd into your planet directory and type 'vagrant up prod'. Once done working, use command 'vagrant halt prod'.

**Comment by ibnahmadbello** [comment]
If you are using Linux distro, you can create an alias for the command if you think typing "vagrant up prod" command is a huge task
